docs/usage.rst

```diff
@@ -95,19 +95,23 @@ To validate a RAML file with Python:
 
    >>> from ramlfications import validate
    >>> RAML_FILE = "/path/to/my-api.raml"
-   >>> validate(RAML_FILE)
-   >>>
+   >>> errors = validate(RAML_FILE)
+   >>> if not errors:
+   >>>     print("Success!")
 
 .. code-block:: python
 
    >>> from ramlfications import validate
    >>> RAML_FILE = "/path/to/invalid/no-title.raml"
-   >>> validate(RAML_FILE)
-   InvalidRootNodeError: RAML File does not define an API title.
+   >>> errors = validate(RAML_FILE)
```

Comment:
I agree that a successful validation should return nothing instead of an empty list.  I'll put that back in.  

Hrmm, if we revert to raising an exception on invalid, how do you propose we report ALL errors that were collected?  Would we create a new Exception class that knows how to work with multiple errors instead of just raising an individual error?